napari/layers/base/base.py

```diff
@@ -734,58 +737,58 @@ def _on_editable_changed(self) -> None:
         """Executes side-effects on this layer related to changes of the editable state."""
 
     @property
-    def scale(self):
-        """list: Anisotropy factors to scale data into world coordinates."""
+    def scale(self) -> npt.NDArray:
+        """array: Anisotropy factors to scale data into world coordinates."""
         return self._transforms['data2physical'].scale
 
     @scale.setter
-    def scale(self, scale):
+    def scale(self, scale: Optional[npt.NDArray]) -> None:
         if scale is None:
-            scale = [1] * self.ndim
+            scale = np.array([1]) * self.ndim
```

Comment:
For `ndim=3` previous version will evaluate to `[1, 1, 1]`, Current will evaluate to `np.ndarray([3])` 

```suggestion
            scale = np.array([1] * self.ndim)
```